.github/CODEOWNERS

```diff
@@ -7,11 +7,12 @@
 # any PR that touches Go files should get a review from go-owners
 #*.go @daos-stack/go-owners
 
-# any PR that touches VOS files should get a review from VOS
-#src/vos/ @daos-stack/vos
-
-# any PR that touches btree files should get a review from VOS
-#src/common/btree*.* @daos-stack/vos
+# Notify vos-watcher of files touched affecting VOS
+src/vos/ @daos-stack/vos-watcher
```

Comment:
It's a nitpick, but it seems like these team names should be pluralized. e.g. vos-watchers, go-watchers, etc.

Would you mind making that update and also adding go-watchers to the line above this one?

Reply:
@mchaarawi would need to update his new team as well

Reply:
Perhaps, we should just merge his PR into this one

Reply:
I'll update them all